smart_open/asb.py

```diff
@@ -0,0 +1,410 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 Nicolas Mitchell <ncls.mitchell@gmail.com>
+#
+# This code is distributed under the terms and conditions
+# from the MIT License (MIT).
+#
+"""Implements file-like objects for reading and writing to/from Azure Storage Blob (ASB)."""
+
+import io
+import logging
+import sys
+
+from azure.storage.blob import BlobServiceClient, StorageStreamDownloader
```

Comment:
Please

```python
import azure.store.blob
```

and then use the fully qualified name in your source, for this and for everything else imported.